src/services/firebase/Firebase.tsx

```diff
@@ -0,0 +1,88 @@
+import firebase from 'firebase'
+import { NFTQUERY } from 'services/Storage/NFT'
+import { NFTS } from 'state/nfts/reducer'
+
+const config = {
```

Comment:
Never include any kind of config in the code base. All these values can differ based on the environment you deploy to (local, dev, qa, prod, etc). So this kind of config should be brought in as an environment variables